CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 ## [next]
 
```

Comment:
We can get rid of this `[next]` piece. This is just a placeholder for when we publish new version.